src/flags/run_time_configurable.cpp

```diff
@@ -54,6 +54,9 @@ DEFINE_double(query_execution_timeout_sec, 600,
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
 DEFINE_bool(cartesian_product_enabled, true, "Enable cartesian product expansion.");
 
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+DEFINE_bool(experimental_text_search_enabled, false, "Enable experimental text search.");
```

Comment:
Put this under `--experimental-enabled=text-search` (https://www.notion.so/memgraph/experimental-enabled-Applied-ddb49ce36df746da8c19679f8e8358e5), as soon as possible.

Reply:
Will do when it's merged

Reply:
Done